packages/graphql/src/schemaTypes/objectTypes/gql-WizardFrontendFramework.ts

```diff
@@ -31,6 +31,10 @@ export const WizardFrontendFramework = objectType({
       description: 'The display name of the framework',
     })
 
+    t.nonNull.boolean('alpha', {
+      description: 'Whether the framework is in alpha or not',
+    })
```

Comment:
Just curious if it's worth codifying this as `supportStatus` at this point. Do we anticipate marking future frameworks as "beta" when component testing overall leaves beta?

Reply:
I'm expecting so - Jess has talked about support for Svelte and is giving a presentation on it soon, and Angular is another one we "support" (inside `npm/angular`, and part of our "official" adapters) but certainly isn't production ready.

Reply:
Done.